integrations/qdrant/src/haystack_integrations/document_stores/qdrant/document_store.py

```diff
@@ -325,6 +517,21 @@ def _query_by_sparse(
         scale_score: bool = True,
         return_embedding: bool = False,
     ) -> List[Document]:
+        """
+        Queries Qdrant using sparse embeddings and returns the most relevant documents.
```

Comment:
```suggestion
        Queries Qdrant using a sparse embedding and returns the most relevant documents.
```

(Nit: This is an internal method and probably did not need to be documented in depth. Better that way, anyway!)